.ci/ppc64le/configuration_ppc64le.yaml

```diff
@@ -19,6 +19,12 @@ test:
 docker:
   Describe:
     - Update CPU set
+    - CPUs and CPU set
```

Comment:
@nitkon did you file an issue for all thesee tests?

Reply:
@devimc : Issue for all these tests? umm..no. Should I for each test skipped?

Reply:
@nitkon no, just one is fine, I would like to track what tests are not working in each architecture, also if you want you can add it to the limitation document

Reply:
@nitkon - if you could add a comment with a reference to the Issue you opened in here, that'd be great, as then later on we can find/fix easier?

Reply:
OK, #nnnn in comment is good enough I think - will merge.